resources/js/Pages/Home.vue

```diff
@@ -132,24 +135,40 @@
 
     export const MAX_NUM_IDS = 600;
 
+    // Run it with compat mode
+    // https://v3-migration.vuejs.org/breaking-changes/v-model.html
+     CdxTextArea.compatConfig = {
+        ...CdxTextArea.compatConfig,
+        COMPONENT_V_MODEL: false,
+    };
+
     export default defineComponent({
         components: {
           CdxDialog,
           CdxButton,
+          CdxField,
           CdxIcon,
           CdxMessage,
-          InertiaHead,
-          TextArea,
+          CdxProgressBar,
+          CdxTextArea,
+          InertiaHead
         },
         setup() {
+            const store = useStore();
+            const textareaInputValue = ref(store.lastSearchedIds);
+            
             return {
                 cdxIconDie,
-                cdxIconInfo
+                cdxIconInfo,
+                textareaInputValue,
+                form: {
+                    itemsInput: store.lastSearchedIds
+                },
```

Comment:
Wait, so do we even still need this now that we have textAreaInputValue?

Reply:
Ah, no, it is not needed. I thought i had removed it.

Reply:
> Wait, so do we even still need this now that we have textAreaInputValue?

Ah, no, it's not needed. I thought I had removed it already.